### ISSUE TITLE
displayActivityIndicator

### ISSUE THREAD

**Issue by nickschafhauser** [alexaubry/BulletinBoard#50]
Is there a way to present the activity indicator without the user having to click on a button?
I am trying to make it where as soon as the bulletin is displayed, the activator indicator immediately appears as well.

Here is my source code to help you understand what I am trying to do:
static func submitQuestionnairePage() -> FeedbackPageBulletinItem {
        
        let page = FeedbackPageBulletinItem(title: "Submitting Questionnaire")
        page.descriptionText = "Survey complete!"
        page.isDismissable = false
        page.manager?.displayActivityIndicator()
       
        return page
        
    }

When I run this code the bulletin does not display the loading icon. It only displays the page title and description text.

**Comment by alexaubry** [comment]
When you create the page like shown in your code sample, the `manager` property is `nil` ; that's why `displayActivityIndicator()` does nothing.

What you should be able to do, is calling `displayActivityIndicator` after preparing the bulletin manager, but before presenting it:

~~~swift
manager.prepare()
manager.displayActivityIndicator()
manager.presentBulletin(above: self)
~~~

Let me know if that works for you.

**Comment by nickschafhauser** [comment]
@alexaubry That didn't seem to make a difference. I am still having the same behavior as I had earlier without that change.

**Comment by alexaubry** [comment]
I've worked on a way to show the activity indicator as soon as the item is displayed, and implemented this API to do it:

- Set `shouldStartWithActivityIndicator` on the item to `true`. The content views will be hidden and the activity indicator will be displayed.

- Implement `onDisplay` or `presentationHandler` to detect when the item is presented. From here, you can perform your work and hide the activity indicator when appropriate.

You can update the library to 2.0.0-beta.1 to test this feature. Please let me know if you encounter any bug, or have any feedback with activity indicators.

**Comment by alexaubry** [comment]
The new behavior shipped in `2.0.0-beta.3`, thanks for the suggestion! 😄 Feel free to reopen this issue if you have any feedback.
